packages/@webex/plugin-meetings/src/reachability/util.ts

```diff
@@ -22,3 +22,27 @@ export function convertStunUrlToTurn(stunUrl: string, protocol: 'udp' | 'tcp') {
 
   return url.toString();
 }
+
+/**
+ * Converts a stun url to a turn url
+ *
+ * @param {string} stunUrl url of a stun server
+ * @param {'tcp'|'udp'} protocol what protocol to use for the turn server
+ * @returns {string} url of a turn server
+ */
+export function convertStunUrlToTurnTls(stunUrl: string, protocol: 'udp' | 'tcp') {
```

Comment:
I guess it doesn't make sense to have the protocol argument here, because you can't have TLS over UDP when talking to a TURN server, right?

Reply:
That seems right, removed.